### ISSUE TITLE
how to set-up meta property="fb:app_id"

### ISSUE THREAD

**Issue by thomas6382** [DerMambo/ms-seo#85]
I tried this way. Only the fb:app_id is not working

    SEO.set({
       title: "title",
       meta: {
                'description': "meta"   
           },
        og: {
          'title': "title",
          'type': "type",
          'url':"url"
        },
       fb: {
          'app_id':"11111111111111"
        }
})

Thanks

**Comment by elie222** [comment]
Yeah. Doesn't look you're able to set `<meta property="fb:app_id" content="appid"/>` through this package at this time.

**Comment by elie222** [comment]
Every time `property=` is set in the code, it's either with twitter or og: https://github.com/DerMambo/ms-seo/blob/master/seo.coffee

**Comment by kenjoe** [comment]
Any update on this? Is `fb:app_id` not yet supported?
